### ISSUE TITLE
Archive repository in light of LibrePhotos?

### ISSUE THREAD

**Issue by jogama** [hooram/ownphotos-backend#159]
Just a drive by comment and observation. Seeing that a repository is archived directs me to check the README for its new home, if any. Feel free to close if it's not what you have in mind.

**Comment by DigitalWarrior** [comment]
Correct. Development of Ownphotos has stopped. The repository was forked and development has continued over at https://github.com/LibrePhotos/librephotos
